crates/server/src/server/server_future.rs

```diff
@@ -14,7 +18,7 @@ use std::{
 use futures_util::{FutureExt, StreamExt};
 #[cfg(feature = "dns-over-rustls")]
 use rustls::{Certificate, PrivateKey};
-use tokio::{net, task::JoinSet};
+use tokio::task::JoinSet;
```

Comment:
Please avoid making unrelated changes in the same commit/PR, this makes it harder to review the changes you've made.